handlers/token.go

```diff
@@ -18,16 +19,31 @@ var (
 )
 
 func init() {
-	currentLeader.Set("127.0.0.1:4001")
+	// etcd cluster is passed in via commandline option -etcd
+	//currentLeader.Set("127.0.0.1:4001")
```

Comment:
each member could handle key RESTful API now, so maybe there is no need to update current leader. You can just pin on one endpoint